src/javaRosa.js

```diff
@@ -792,7 +792,12 @@ define([
     };
 
     var itextMediaBlock = function (mug, options) {
-        var block = itextConfigurableBlock(mug, options);
+        var block = itextConfigurableBlock(mug, options),
+            pathPrefix = options.pathPrefix;
+
+        if(!_.isString(options.pathPrefix)) {
```

Comment:
Is this another case of detecting null or undefined? I'm finding this pattern of using `_.isString` instead of `_.isUndefined` or `=== null` to raise more questions than in answers, but maybe I'm being silly?